about/history.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: History - XRPL.org
+redirectFrom:
+  - /history.html
+---
+
+import History from "../_components/pages/History";
+import { EmptyLayout } from "@redocly/developer-portal/ui";
+
+export default EmptyLayout;
+
+<History />
```

Comment:
This needs to be wrapped in a `LayoutWrapper` so that it doesn't get pushed under the top nav.

```suggestion
<LayoutWrapper>
  <History />
</LayoutWrapper>
```

> **Important:** You also have to import `LayoutWrapper` like this:
> 
>     import LayoutWrapper from '../_components/LayoutWrapper';